### ISSUE TITLE
Definitely need to be able to admin the georeferenced photos!

### ISSUE THREAD

**Issue by mpdavies** [layersoflondon/application#126]
Some horrifically bad georeferencing happening .... which partly means that people do not have the help they need (so partly a Klokan issue, partly something for us), but also that we must have a way for us to administer uploads, whether pins or photos. Ideally the team need a way to go in an delete photos and pins - and in the case of deleted photos they need to be restored to the 'to do' photos in Georeferencer.

**Comment by edtjones** [comment]
I think we need to use Georeferencer's moderation tools @mpdavies - it's not sustainable to have to flag and fix every one is it :( Will discuss with them tomorrow.

**Comment by mpdavies** [comment]
Yes, good idea: we'll add that to the list of things we can try and get done by Klokan in this phase.
